analyzers/utils.go

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
Please rename the file to libalalyzers so we can be consistent with other services.